src/components/iptables/scripts/pcn-iptables-init-xdp

```diff
@@ -3,26 +3,21 @@
 # check if polycubed is alive
 polycubed_alive=$(ps -el | grep polycubed)
 if [ -z "$polycubed_alive" ]; then
-    echo "polycubed is NOT running."
-    echo "please launch it using ..."
-    echo "'sudo polycubed'"
+    echo "pcn-iptables is not able to find any daemon running."
+    echo "please try to run:"
+    echo ""
+    echo "sudo polycubed"
     exit 1
-    # sudo polycubed
-else
-    echo "polycubed already launched ..."
 fi
 
 ret=$(polycubectl iptables pcn-iptables show)
 ret=$(echo $?)
 
 if [ $ret -ne 0 ]
     then
-        echo "Launching pcn-iptables ..."
+        echo "launching pcn-iptables ..."
```

Comment:
`Line 6` mentions `running`, to maintain the uniformity should we say:
`"Starting pcn-iptables"` and `"pcn-iptables already running"`

What say @mbertrone ?

Reply:
Agree.